PHPCI/Plugin/Pdepend.php

```diff
@@ -102,7 +102,17 @@ public function execute()
         $config = $this->phpci->getSystemConfig('phpci');
 
         if ($success) {
```

Comment:
I have no idea how to fix this, there's only one new line after "if ($usccess) {".
Anyone knows why this is happening?
Thanks